src/Microsoft.ML.Data/DataLoadSave/Text/TextLoaderSaverCatalog.cs

```diff
@@ -144,20 +148,22 @@ public static IDataView ReadFromTextFile(this DataOperationsCatalog catalog, str
         /// <param name="headerRow">Whether to write the header row.</param>
         /// <param name="schema">Whether to write the header comment with the schema.</param>
         /// <param name="keepHidden">Whether to keep hidden columns in the dataset.</param>
+        /// <param name="forceDense">Whether to save columns in dense format even if they are sparse vectors.</param>
         public static void SaveAsText(this DataOperationsCatalog catalog,
             IDataView data,
             Stream stream,
             char separatorChar = TextLoader.DefaultArguments.Separator,
```

Comment:
Should this be` TextSaver.DefaultArguments.Separator`?

Reply:
Actually we should match the signature of `TextLoader` which uses a separatorChar. If this logic works, then my comment about the need to use "tab" instead of `'\t'.ToString()` was wrong

---
In reply to: [259067133](https://github.com/dotnet/machinelearning/pull/2630#discussion_r259067133) [](ancestors = 259067133)

Reply:
If it didn't work, some tests may fail. I agree char is better.

---
In reply to: [259071246](https://github.com/dotnet/machinelearning/pull/2630#discussion_r259071246) [](ancestors = 259071246,259067133)